delete.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+session_start();
+require_once 'functions.php';
+
+$id = targetID();
+$db = getDb();
+removeData($db, $id);
```

Comment:
no check to confirm if the deletion worked properly, also this will just display a blank page to the user when they press delete. id recommend redirecting them back to the collection page

Reply:
the redirect is in functions.php